### DISCUSSION TITLE
503 Service Temporarily Unavailable

### DISCUSSION THREAD

**Discussion by budiaman** [kirimwa-id/forum#146] [Tanya Jawab] [answered]
Selamat sore mas Rio,
Saya mau coba post device, untuk sekarang apakah ada masalah ya mas.
Statusnya seperti ini mas : **503 Service Temporarily Unavailable**

Bisa bantu info mas, terkait kendala tersebut
![Error-503](https://user-images.githubusercontent.com/11144010/198260689-03663067-adc8-48dc-a760-f3c508fb6ba8.jpg)

**Comment by rioastamal** [comment, marked as answer]
27 Oktober sore sekitar pkl 17:00 memang ada masalah di server tapi tidak lama kemudian harusnya udah kembali normal.

**Reply by budiaman** [reply]
Baik mas, untuk sekarang saya cba sudah bisa mas Rio.
Terima kasih mas Rio.
